api/src/main/java/org/pmiops/workbench/api/OfflineEnvironmentsController.java

```diff
@@ -53,8 +54,8 @@
  * Workbench service account.
  */
 @RestController
-public class OfflineRuntimeController implements OfflineRuntimeApiDelegate {
```

Comment:
this class was not strictly related to *runtimes* but also apps and disks, hence the rename